src/controller/loot_table.controller.ts

```diff
@@ -0,0 +1,121 @@
+import { UpdatedAt } from 'sequelize-typescript';
+import { LootTable } from '../models/loot_table.model.js'
+import { createPayload } from './types/loot_table_types.js';
+
+
+///////Read or Pull all
+export const findAllLootTable = async (req, res) => {
+    try {
+        console.log("Calling find all for Loot on the Table")
+        const allLootTables:LootTable[] = await LootTable.findAll();
+        res.status(200).json({message: allLootTables})
+    } catch (error) {
+       res.status(500).json({ message: error.message });
+    }
+}
+
+//Pull a SINGLE PERSON NEEDS TO BE FIXED
+// export const findSingleLootTable = async (req, res) => {
+//     try {
+//         console.log("Calling find all for Static Mates")
+// //         const allLootTables:LootTable[] = await LootTable.findAll();
+// //         res.status(200).json({message: allLootTables})
+//     } catch (error) {
+//         res.status(500).json({ message: error.message });
+//     }
+// }
+
+///create
+
+export const createSingleLootTable = async (req, res) => {
+    try {
+        //properties on the query object and the query object is property on the request object 
+        const createPayload:createPayload = req.body;
+        console.log("req.body", req.body)
+        //find a single pesron
+        const findSingleLootTable = await LootTable.findOne({ where: {name_of_gear: createPayload.name_of_gear } });
+        if (findSingleLootTable){
+            throw Error("There was already loot piece with that name")
+            return
+        }
+        //this is the set up of paramaters for postman
+        const single_loot_piece = await LootTable.create({
+                piece_type: createPayload.piece_type,
+                name_of_gear: createPayload.name_of_gear,
+                floor_id: createPayload.floor_id,
+                gear_source: createPayload.gear_source,
+                iLvl: createPayload.iLvl,
+                createdAt: new Date(),
+                updatedAt: new Date(),
+        })
+        //this is confirming that things worked
+        if (single_loot_piece){
+            console.log("loot piece was created", single_loot_piece.dataValues)
+        }
+        else {
+            throw Error("something went wrong")
+        }
+        res.status(200).json({message: "Success", data: single_loot_piece});
```

Comment:
Take a look at how I did the message, also **create a return type**

Reply:
```
    return res.status(200).json({ message: 
      {
          status: 'Sucesss',
          data: 'The following was created:' + single_loot_piece
      }
```

This looks about right